drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -1852,7 +1865,8 @@ int bnxt_hwrm_allocate_vfs(struct bnxt *bp, int num_vfs)
 		add_random_mac_if_needed(bp, &req, i);
 
 		HWRM_PREP(req, FUNC_CFG, -1, resp);
-		req.fid = rte_cpu_to_le_16(bp->pf.first_vf_id + i);
+		req.flags = rte_cpu_to_le_32(bp->pf.vf_info[i].func_cfg_flags);
```

Comment:
I don't see the vf_info.func_cfg_flags being set anywhere.
Is this more for future?